### PR TITLE
added card component

### PR DESCRIPTION
**Pull request by PeeyushPrashant**

### CODE REVIEW

File: Components/Card/card.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  </head>
+  <body>
+    <div class="card-container">
+      <!--Card with badge-->
+
+      <div class="card">
+        <div>
+          <section class="relative">
+            <span class="absolute card-badge">Insane</span>
+            <img src="/asset/image/apdhillon.jpg" alt="" />
```
Comment (anuj-kumary) [inline Components/Card/card.html]: Always give some value to **alt**.

**Review by anuj-kumary — CHANGES_REQUESTED** [review]
In horizontal card give the border-radius to the whole image so it looks NYC also there are so many whitespaces so remove that space.

**Review by Rohanmond — APPROVED** [review]
Just fix the horizontal one. After that you are good to go.

**Review by vinc3nati — APPROVED** [review]
Changes are good, proceed!

**Pull request merged**